recipe/meta.yaml

```diff
@@ -11,38 +10,44 @@ source:
   sha256: 8948a35f6f85805c8ac36e94d5492c86a34c39dcf6f405b0f84491bc444e3479
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [py<39]
 
 requirements:
   host:
-    - python {{ python_min }}
+    - python
     - hatchling
     - pip
   run:
+    - python
     - pydantic >=2.7.4
     - python-xxhash >=3.5.0
     - langgraph-prebuilt >=0.2.0
     - langgraph-sdk >=0.1.42
     - langgraph-checkpoint >=2.0.26
-    - python >={{ python_min }},<4.0
     - langchain-core >=0.1
 
 test:
   imports:
     - langgraph
   commands:
     - pip check
+    - python -c "from importlib.metadata import version; assert(version('{{ name }}')=='{{ version }}')"
```

Comment:
Could you check upstream tests locally (after locally solving the circular dependency)?
https://github.com/langchain-ai/langgraph/tree/0.4.7/libs/langgraph/tests